core/modules.rs

```diff
@@ -484,20 +484,14 @@ impl Deps {
     }
   }
 
-  pub fn to_json(&self) -> String {
-    let mut children = "[".to_string();
-
-    if let Some(ref deps) = self.deps {
-      for d in deps {
-        children.push_str(&d.to_json());
-        if !d.is_last {
-          children.push_str(",");
-        }
-      }
+  pub fn to_json(&self) -> serde_json::Value {
```

Comment:
This to_json() function was not used.
It was originally written in https://github.com/denoland/deno/pull/2223.
I rewrote it to build a JSON with serde instead of writing the string manually, it was easier for me since I put it later in a bigger JSON object.
I did not change the shape of the JSON, it's the same.

Reply:
Great!